### PR TITLE
The direction's travel mode button is highlighted when clicked or selected

### PR DESCRIPTION
**Pull request by sharon-ho**
When getting directions, the selected travel mode's icon is highlighted.

### CODE REVIEW

File: app/src/main/java/com/example/campusguide/Constants.kt [hunk 1]
```diff
@@ -36,4 +36,7 @@ object Constants {
         45.458153, -73.640490
     )
 
-}
\ No newline at end of file
+    const val PRIMARY_COLOR = "#008577"
+    const val PRIMARY_COLOR_DARK = "#00574B"
+    const val ACCENT_COLOR = "#D81B60"
```
Comment (pobudzey) [inline app/src/main/java/com/example/campusguide/Constants.kt:41]: These colors are already defined in the colors resource file, maybe we could use those?

**Review by pobudzey — APPROVED** [review]
Works as expected!

File: app/src/main/java/com/example/campusguide/Constants.kt (hunk 1 above)
Comment (sharon-ho) [reply app/src/main/java/com/example/campusguide/Constants.kt:41]: I forgot you could access them from the XML using `R.color` so I will use the values defined in the colors resource file in that case!

Comment (sharon-ho) [reply app/src/main/java/com/example/campusguide/Constants.kt:41]: Actually never mind, I tested out `R.color` and it crashes so I'm not sure if it's possible. For now I'll add them as constant values in `Constants.kt` and if I find a way to reference the colors from the XML then I'll do that.

**Pull request merged**